.github/workflows/ci.yml

```diff
@@ -290,3 +290,36 @@ jobs:
         with:
           name: notebook-outputs
           path: out
+  ts-lint:
+    name: Typescript lint check
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          node-version: '14.16.1'
+      - name: Install npm
+        run: npm --prefix 'cirq-web/cirq_ts' ci
+      - name: Lint
+        run: check/ts-lint 
+  ts-test:
+    name: Typescript unit testing
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          node-version: '14.16.1'
+      - name: Install npm dependencies
+        run: npm --prefix 'cirq-web/cirq_ts' ci
```

Comment:
refactor `npm --prefix 'cirq-web/cirq_ts'` to a new script `check/npm` which forwards all the arguments and replace the original call in all the scripts, so that in case the prefix changes, we can change it in a single place.